feat(curriculum): add password generator project to Python curriculum (#52444)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/65648e4c5b316c4ec5e4fddc.md

---
id: 65648e4c5b316c4ec5e4fddc
title: Step 15
challengeType: 20
dashedName: step-15
---

# --description--

> At the bottom of your function, create a `password` variable and assign it an empty string.

Reviewer: ```suggestion
At the bottom of your function, declare a `password` variable and assign an empty string to this variable.
```
